src/explicitode.jl

```diff
@@ -265,13 +286,28 @@ function taylorinteg{T<:Number}(f, x0::T, t0::T, t_max::T,
     return view(tv,1:nsteps), view(xv,1:nsteps)
 end
 
-function taylorinteg{T<:Number}(f, q0::Array{T,1}, t0::T, t_max::T,
-        order::Int, abs_tol::T; maxsteps::Int=500)
+function taylorinteg{S<:Number, T<:Number, U<:Number, V<:Number}(f,
+        q0::Array{S,1}, t0::T, tmax::U, order::Int, abstol::V; maxsteps::Int=500)
+
+    #promote to common type before integrating:
```

Comment:
Lines 294-295, and 298: Isn't it easier to write one single line here? For instance,
```julia
elq0, t0, tmax, abstol = promote(q0[1], t0, tmax, abstol)
```
instead of the three lines?